src/UI/class-metadata-header.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Parse.ly header metadata class
+ *
+ * @package Parsely
+ * @since 3.3.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\UI;
+
+use Parsely\Metadata;
+use Parsely\Parsely;
+use WP_Post;
+
+use const Parsely\PARSELY_FILE;
```

Comment:
Thoughts on injecting this into the constructor to avoid a reference to a global constant? It's essentially used to ask "What is the path to the `views/` directory?".

Reply:
I like it. This way it's easier for us to test too.

Reply:
I can actually see both points. I think consistency is important, and indeed doing the change here would mean adopt the same pattern somewhere else, which might be tricky.